gencert-env.private.example.sh

```diff
@@ -0,0 +1,8 @@
+COMMON_NAME=Comune di Roma
+ENTITY_ID=https://spid.comune.roma.it
+KEY_LEN=3072
+LOCALITY_NAME=Roma
+MD_ALG=sha256
+ORGANIZATION_IDENTIFIER=VATIT-02438750586
+ORGANIZATION_NAME=Comune di Roma
+SPID_SECTOR=private
```

Comment:
Why not naming it `private.env.example`?

Reply:
for editor highlighting: having it with the `.sh` extension automatically use the correct one, with `.example` you have to set it manually every time.

Reply:
Local editor behavior/settings should not influence the source code. I would prefer to name it as suggested.